### PR TITLE
CONSOLE-2361: Remove orphaned Bootstrap 3 input-group styles

### PR DESCRIPTION
**Pull request by rhamilto**
The instances of input-group in use were migrated to PatternFly 4, but these Bootstrap 3 styles were orphaned.

### CODE REVIEW

**Comment by rhamilto** [comment]
/hold for approval

/assign @yapei

@yapei We just need a regression test to make sure there are no visual problems with these changes.

**Comment by spadgett** [comment]
This PR has no user visible changes, and we have general docs and px approval for this epic. @ahardin-rh @sferich888 FYI

/label docs-approved
/label px-approved
/hold cancel

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**
